pkg/cloudevents/transport/http/codec_v02.go

```diff
@@ -78,13 +78,18 @@ func (v CodecV02) encodeBinary(e cloudevents.Event) (transport.Message, error) {
 		return nil, err
 	}
 
-	mediaType, err := e.Context.GetDataMediaType()
+	//mediaType, err := e.Context.GetDataMediaType()
```

Comment:
Any reason to comment out rather than remove?

Reply:
🤕 this was a hard refactor tbh